### ISSUE TITLE
alwayson-ag-db-rpo-rto should be horizontalbar chart

### ISSUE THREAD

**Issue by Matticusau** [Matticusau/sqlops-widgets#5]
The *alwayson-ag-db-rpo-rto* widget doesn't fit too well when running as a Bar due to the labels.

Currently it is defined as:

```
"type": {
                        "bar": {
                            "dataDirection": "vertical",
                            "dataType": "number",
                            "legendPosition": "top",
                            "labelFirstColumn": false,
                            "columnsAsLabels": true
                        }
                    },
```

This should be changed to be:

```
"type": {
                "horizontalBar": {
                    "dataDirection": "vertical",
                    "dataType": "point",
                    "legendPosition": "bottom",
                    "labelFirstColumn": false,
                    "columnsAsLabels": true
                }
            }
```

**Comment by Matticusau** [comment]
Size might also need to be adjusted

**Comment by Matticusau** [comment]
This has been released in v0.2.0.0 #10
